manual/manual/macros.tex

```diff
@@ -205,4 +205,25 @@
 
 \newenvironment{maintitle}{\begin{center}}{\end{center}}
 
+
+
+% Caml-example related command
+\newenvironment{camlexample}[1]{
+  \ifnum\pdfstrcmp{#1}{toplevel}=0
+    \renewcommand{\hash}{\#}
+  \else
+    \renewcommand{\hash}{}
+  \fi
+}{}
+\newenvironment{caml}{}{}
+\newcommand{\ocamlkeyword}{\bfseries}
+\newcommand{\ocamlhighlight}{\bfseries\uline}
+\newcommand{\ocamlerror}{\bfseries}
+\newcommand{\ocamlwarning}{\bfseries}
+\newcommand{\ocamlcomment}{\normalfont\small}
+\newcommand{\ocamlstring}{\bfseries}
```

Comment:
The LaTeX styling proposed here is meh, but I guess we can iterate on this later. I would find it nicer to correspond to the HTML styling (comments in gray for example), and I believe that I've used colors in lstlistings before (I don't remember exactly how).

Reply:
lstlistings supports colors of course. See a couple examples:
- https://texblog.org/2011/06/11/latex-syntax-highlighting-examples/
- https://stackoverflow.com/questions/1304315/highlighting-a-chunk-of-code-within-a-lstlisting

Reply:
Using color is quite straightforward indeed. And using grey should not affect black-and-white readability.